README.md

```diff
@@ -1,16 +1,94 @@
+  Any commits made will be associated with the user who created them, and
+  GitHub allows for granular, role-based access that can also be revoked
+  when someone departs CISA.
+- How do I use GitHub? Where do I start?
+  - [18F Intro to GitHub](https://handbook.tts.gsa.gov/intro-to-github/)
+  - [Digital.gov intro to GitHub](https://digital.gov/resources/an-introduction-github/)
+  - Always remember that GitHub is a version control system, so all changes
+  are saved, so there is nothing we can't fix or undo.
+
+### GitHub access ###
+
+- How do I get access to the `cisagov` organization?
+  - For federal employees, send your GitHub username to github@cisa.gov
+  - For contractors, ask your fed lead to send in your username
+- How do I get put on a team within the `cisagov` organization?
+  - See [Github's organizations and teams documentation](https://docs.github.com/en/organizations)
+  - You'll need to contact a team member or a `cisagov` administrator
+- How do I get access to a specific repository?
+  - Most repositories are associated with the [teams](https://docs.github.com/en/organizations/organizing-members-into-teams)
+  that use them, so to get access you can contact github@cisa.gov, a
+  team member, or a `cisagov` administrator to get access
+- How do I make a new repository?
+  - [GitHub's documentation on creating a repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-new-repository)
+  - [`cisagov`'s Project Setup guide](https://github.com/cisagov/development-guide/tree/develop/project_setup)
+
+## Policies and content guides ##
+
+- What belongs in `cisagov` versus another GitHub organization?
+  - Here are some questions to ask when considering posting a project:
+    - Does CISA use or develop the software? Is it developed by or for one of
+    the groups or divisions within CISA?
+      - If not, we recommend the authors create their own GitHub organization
+      and post their work there
+    - What source control system is in place currently?
+      - Many source control systems, such as Mercurial and GitLab, can export
+      the entire development history for import into GitHub - this is by far
+      the preferred method
+    - Is the project still under active development or is it in maintenance?
+      - If the project no longer has a team performing maintenance, we
+      recommend the repository be archived to make that clear to people who
+      may want to use it
+- What belongs on [cisa.gov](https://www.cisa.gov) versus on `cisagov`?
+  - The [cisa.gov](https://www.cisa.gov) site is primarily focused at an
+  audience outside of CISA, such as Critical Infrastructure partners or the
+  public
+  - `cisagov` is for both internal and external users, as well as partners. It exists
+  specifically to share projects with the public as well as internal users.
+- Working in the public (dos and don'ts, best practices)
```

Comment:
Take or leave this one, but to my eyes, it works better this way:
```suggestion
- Working in public (dos and don'ts, best practices)
```

Reply:
Looks great!